File: inappchat/src/main/java/com/livechatinc/inappchat/models/Author.java
package com.livechatinc.inappchat.models;

import androidx.annotation.NonNull;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Author {
    @SerializedName("name")
    @Expose
    private String name;

    public String getName() {
        return name;
    }

    @NonNull
    @Override
    public String toString() {
        return "Author{" +
                "name='" + name + '\'' +
                '}';
    }
}

File: ChatWidget/src/main/java/com/livechatinc/chatwidget/src/models/Author.kt
package com.livechatinc.chatwidget.src.models

import kotlinx.serialization.Serializable

@Serializable
data class Author(
    val name: String? = null
) {
    override fun toString(): String {
        return "Author{" +
                "name='" + name + '\'' +
                '}'
    }
}
